src/test/scala/com/cognite/sdk/scala/v1/EventsTest.scala

```diff
@@ -13,6 +13,7 @@ class EventsTest extends SdkTestSpec with ReadBehaviours with WritableBehaviors
   private val idsThatDoNotExist = Seq(999991L, 999992L)
   private val externalIdsThatDoNotExist = Seq("5PNii0w4GCDBvXPZ", "6VhKQqtTJqBHGulw")
 
+  // Unpartitioned read is too slow.
```

Comment:
General statement, or meant to explain something?

Reply:
I think that's why it's commented out, right? I want to explain the reason.

Reply:
Ahh, I got confused by the neat syntax.. I thought this was an existing comment, not commented out code hehe
`  // it should behave like readable(client.events)`